<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ordenation</class>
 <widget class="QDialog" name="Ordenation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rsc.qrc">
    <normaloff>:/icones/Images/Icones/iconeSort.png</normaloff>:/icones/Images/Icones/iconeSort.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutScreen">
   <item>
    <widget class="QWidget" name="widgetInicio" native="true">
     <widget class="QGroupBox" name="groupBoxInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Input</string>
      </property>
      <widget class="QRadioButton" name="radioButtonInGerar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Gerar</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonInCarregar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Carregar do .txt</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxParametros">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>531</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Parâmetros</string>
      </property>
      <widget class="QPushButton" name="pushButtonGerar">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Gerar</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>411</width>
         <height>45</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayoutParametros">
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditQtdeEle">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelValorInicio">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Valor de início</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEditValueBegin">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelOrdem">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Ordem</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelQtdeElement">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Qtde de elementos</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboBoxOrder">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Crescente</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Decrescente</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Aleatório</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxOrdenar">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>0</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Ordenação</string>
      </property>
      <widget class="QPushButton" name="pushButtonOrdenar">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ordenar</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>136</width>
         <height>45</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayoutOrdenation">
        <item row="0" column="0">
         <widget class="QLabel" name="labelAlgoritmos">
          <property name="text">
           <string>Selecione os algoritmos</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonAlgoritmos">
          <property name="text">
           <string>Todos</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayoutAlgoritmos">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelTempo">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-top-color: rgb(0, 0, 0);
border-right-color: rgb(0, 0, 0);
border-left-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Tempos</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelVetores">
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Vetores</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="tableWidgetAlgoXTemp"/>
     </item>
     <item row="1" column="1">
      <widget class="QTableWidget" name="tableWidgetVectores"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelLogs">
     <property name="font">
      <font>
       <family>Arial</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Logs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditLog"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonInGerar</tabstop>
  <tabstop>radioButtonInCarregar</tabstop>
  <tabstop>comboBoxOrder</tabstop>
  <tabstop>lineEditQtdeEle</tabstop>
  <tabstop>lineEditValueBegin</tabstop>
  <tabstop>pushButtonGerar</tabstop>
  <tabstop>tableWidgetAlgoXTemp</tabstop>
  <tabstop>tableWidgetVectores</tabstop>
  <tabstop>pushButtonOrdenar</tabstop>
 </tabstops>
 <resources>
  <include location="../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
